test/common/services/security/test_user.ts

```diff
@@ -77,10 +77,9 @@ export async function createTestUserService(ctx: FtrProviderContext, role: Role,
   const config = ctx.getService('config');
 
   const enabled =
-    !config
-      .get('esTestCluster.serverArgs')
-      .some((arg: string) => arg === 'xpack.security.enabled=false') &&
-    !config.get('security.disableTestUser');
+    !(config.get('esTestCluster.serverArgs') || []).some(
```

Comment:
`config.get('esTestCluster.serverArgs')` can be `undefined` if not specified in FTR config file